config/rootcompute/kube-1.24/apiexport-kubernetes.k8s.io.yaml

```diff
@@ -4,6 +4,7 @@ metadata:
   annotations:
     bootstrap.kcp.dev/create-only: "true"
     bootstrap.kcp.dev/battery: root-compute-workspace
+    extra.apis.kcp.dev/compute.workload.kcp.dev: ""
```

Comment:
empty string is strange.

Reply:
yeah, at some it was a label, so it was less strange I assume.
We can decide to put `"true"` or `"false"`, but I don't see what it brings.
BTW there is a precedent it seems: see https://github.com/kcp-dev/kcp/blob/main/pkg/reconciler/workload/namespace/namespace_reconcile_placementbind.go#L59. In the code it nevers contains anything else than "" or nil (== the annotation is not there)

Reply:
"true" is common

Reply:
done